connect/runtime/src/main/java/org/apache/kafka/connect/runtime/SinkConnectorConfig.java

```diff
@@ -27,13 +28,20 @@
 
 public class SinkConnectorConfig extends ConnectorConfig {
 
-    public static final String TOPICS_CONFIG = "topics";
-    private static final String TOPICS_DOC = "";
+    public static final String TOPICS_CONFIG = SinkTask.TOPICS_CONFIG;
+    private static final String TOPICS_DOC = "List of topics to consume, separated by commas";
     public static final String TOPICS_DEFAULT = "";
     private static final String TOPICS_DISPLAY = "Topics";
 
+    private static final String TOPICS_REGEX_DOC = "Regular expression giving topics to consume." +
+        "<p/>Under the hood, the regex is compiled to a <code>java.util.regex.Pattern</code>. " +
```

Comment:
This sentenceabout Pattern is pulled verbatim from the docs for RegexRouter for consistency.

Reply:
Is the `<p/>` necessary?

Reply:
This piece with the `<p/>` is directly copied from https://github.com/apache/kafka/blob/trunk/connect/transforms/src/main/java/org/apache/kafka/connect/transforms/RegexRouter.java#L30-L32, but I can see how Transforms documentation isn't rendered in the same tabular format. Will remove.